bins/revme/src/cmd/bench/evm_build.rs

```diff
@@ -0,0 +1,12 @@
+use criterion::Criterion;
+use revm::{Context, ExecuteEvm, MainBuilder, MainContext};
+
+pub fn run(criterion: &mut Criterion) {
+    let mut evm = Context::mainnet().build_mainnet();
+
+    criterion.bench_function("evm_build", |b| {
+        b.iter(|| {
+            let _ = evm.replay().unwrap();
```

Comment:
```suggestion
           let _ = Context::mainnet().build_mainnet();
```

Buildering of EVM should be here, not a replay.

Reply:
gotcha! updated to https://github.com/bluealloy/revm/blob/e537c96d8c224603eeccb2bfb2a62407bf5de7a2/bins/revme/src/cmd/bench/evm_build.rs#L4-L9